### DISCUSSION TITLE
clangd Unknown arguments - how to get rid of errors

### DISCUSSION THREAD

**Discussion by edrobertsrayne** [LazyVim/LazyVim#2241] [Q&A] [answered]
When I load a C/C++ file in Lazyvim, clangd loads just fine but I get errors at the top of every file.

```
Unknown argument '-mlongcalls'; did you mean '-mlong-calls'? [drv_unknown_argument_with_suggestion]
Unknown argument: '-fstrict-volatile-bitfields' [drv_unknown_argument]
Unknown argument: '-fno-shrink-wrap' [drv_unknown_argument]
Unknown argument: '-fno-tree-switch-conversion' [drv_unknown_argument]
```

Does anyone have any suggestions about how I can get rid of these?

**Comment by dpetka2001** [comment, marked as answer]
Probably your project has some compile flags that are not compatible with clangd. 
You could try creating a `.clangd` file at the root of your project with contents
```
CompileFlags:
  Remove: [-fconserve-stack, -fno-allow-store-data-races]  <--  Put here the flags that throw errors
```
and see if that helps out. I don't know of a better way and maybe you should ask in the clangd repo instead if 
what I suggested doesn't work for you.
